Sming/Libraries/ThingSpeak/README.md

```diff
@@ -0,0 +1,61 @@
+# Sming - ThingSpeak driver libary
```

Comment:
The Sming API documentation specifically excludes upstream libraries. Such libraries should be appropriately documented but we avoided imposing Sming coding style on external projects. We may want to include the libraries documentation in a Sming API documentation module in the future and having consistent coding style is a bonus. I do not see libraries as part of the core Sming code.